static/src/javascripts/projects/common/modules/commercial/user-features.ts

```diff
@@ -46,15 +38,39 @@ const ARTICLES_VIEWED_OPT_OUT_COOKIE = {
 	daysToLive: 90,
 };
 
-const CONTRIBUTIONS_REMINDER_SIGNED_UP = {
-	name: 'gu_contributions_reminder_signed_up',
-	daysToLive: 90,
-};
+const AD_FREE_USER_COOKIE = 'GU_AF1';
 
 // TODO: isn’t this duplicated from commercial features?
```

Comment:
Could this comment be updated? `forcedAdFreeMode` is now duplicated here: https://github.com/guardian/commercial/blob/b769d8304ab43033466b27e7a580c4e8a861587f/src/lib/user-features.ts

Reply:
so the commercial user-features only runs on DCR https://github.com/guardian/commercial/blob/b769d8304ab43033466b27e7a580c4e8a861587f/src/standalone.commercial.ts#L128 And this one runs on frontend so both are needed at the moment